src/kde/kde.i.hh

```diff
@@ -18,48 +18,113 @@ namespace rtt_kde {
 //------------------------------------------------------------------------------------------------//
 /*!
  * \brief
- * DEFAULT reconstruction to return error if instantiation is not found
+ * epan_kernel basis function used during reconstruction
  *
- * \tparam coord enumerated value to specify KDE coordinate system
- * \tparam dim integer specifying the data dimensionality 
- * \param[in] distribution 
- * \param[in] position
- * \param[in] one_over_band_width
- * \param[in] domain_decomposed
- * \return final local function distribution
+ * Epanechnikov kenrel to be used in reconstrtuction
  *
+ * \param[in] x from kernel origin
+ * \return distribution weight based on distance from the kernel center 
+ *
+ * Test of kde.
  */
+inline double kde::epan_kernel(const double x) const {
+  const double x2 = x * x;
+  return x2 > 1.0 ? 0.0 : 0.75 * (1.0 - x2);
+}
 
-template <int coord>
-template <int dim>
-std::vector<double>
-kde<coord>::reconstruction(const std::vector<double> & /*distribution*/,
-                           const std::vector<std::array<double, 3>> & /*position*/,
-                           const std::vector<std::array<double, 3>> & /*one_over_band_width*/,
-                           const bool /*domain_decomposed*/) const {
-
-  Insist(false, "kde::reconstruction has not been implemented for this coordinate system and or "
-                "dimension combination");
-  return std::vector<double>(1, 0.0);
+//------------------------------------------------------------------------------------------------//
+/*!
+ * \brief
+ * log_transform pushes the distribution to log space which is less sensitive to sharp derivatives.
+ *
+ *
+ * \param[in] value of original distribution
+ * \param[in] bias used to ensure positivity
+ *
+ * \return the logarithmic transform of the original value
+ *
+ * Test of kde.
```

Comment:
I'm not sure this description is accurate.

Reply:
fixed.